ci/pipeline.yaml

```diff
@@ -233,3 +248,64 @@ jobs:
                 ! diff --context=3 \
                   <(tar xzfO nginx-chart/*.tgz nginx/values.yaml) \
                   <(tar xzfO rewritten-chart/chart.tgz nginx/values.yaml)
+
+
+  - name: test-pull-request
+    serial: true
+    plan:
+      - in_parallel:
+        - get: test-image
+          passed:
+            - build-test-image
+        - get: source
+          resource: pull-requests
+          trigger: true
+          version: every
+          params:
+            integration_tool: rebase
```

Comment:
So I guess that this will be rebasing the PR on top of main?

Reply:
Correct